src/entries/background/handlers/handleOpenExtensionShortcut.ts

```diff
@@ -1,13 +1,22 @@
 import { event } from '~/analytics/event';
 import { queueEventTracking } from '~/analytics/queueEvent';
+import { RainbowError, logger } from '~/logger';
 
 /**
  * Handles keyboard commands for the extension
  */
 export const handleOpenExtensionShortcut = () => {
   // firefox maps chrome > browser, but it does still use
   // `browserAction` for manifest v2 & v3. in v3 chrome uses `action`
-  const openPopup = () => (chrome.action || chrome.browserAction).openPopup();
+  const openPopup = () => {
+    try {
+      (chrome.action || chrome.browserAction).openPopup();
+    } catch (error) {
+      logger.error(new RainbowError('Error opening extension popup'), {
+        error,
+      });
+    }
+  };
```

Comment:
potentially the source of this: https://rainbow-me.sentry.io/issues/6504001159/?environment=production&project=4504012577112064&query=is%3Aunresolved%20issue.priority%3A%5Bhigh%2C%20medium%5D&referrer=issue-stream&stream_index=1